denops_std/function/cursor_test.ts

```diff
@@ -46,6 +46,8 @@ test({
         assertNumber(await cursor.line(denops, "."));
         assertEquals(await cursor.line(denops, "."), 1);
         assertEquals(await cursor.line(denops, "$"), 3);
+        assertNumber(await cursor.line(denops, ".", 1000));
+        assertEquals(await cursor.line(denops, ".", 1000), 1);
```

Comment:
*should*

We have no guarantee that the current winid is `1000`. We need to get a correct one first.

Reply:
Fixed.
I changed to use `bufwinid("%")` to get current winid.